utils/mixin.py

```diff
@@ -0,0 +1,38 @@
+from itertools import chain
+
+class DefaultConfigMixin():
+
+    @property
+    def _default_config(self):
+        if (self.bot_config.DEFAULT_CONFIG and self.name
+            in self.bot_config.DEFAULT_CONFIG):
```

Comment:
lines with an `if` clause should be indented four additional spaces so that do not align with the statements in the indented block